benches/serde.rs

```diff
@@ -152,6 +153,23 @@ fn make_big_record() -> (Schema, Value) {
     (big_schema, big_record)
 }
 
+fn make_big_json_record() -> serde_json::Value {
```

Comment:
Would you mind moving the json benchmark into a separate file? It would keep this one much tidier.

Reply:
Sure thing - Check out the latest commit. Thanks